src/lib/tokenizers/alpheios-remote-tokenizer.js

```diff
@@ -0,0 +1,75 @@
+import { ClientAdapters } from 'alpheios-client-adapters'
+
+export default class AlpheiosRemoteTokenizer {
+  /**
+     * Starting point for tokenize workflow
+     * @param {SourceText} docSource - docSource for tokenize
+     * @param {String} idPrefix - prefix for creating tokens idWord
+     * @returns {[Objects]} - array of token-like objects, would be converted to Tokens outside
+     */
+  static async tokenize (docSource, idPrefix, tokenizeParams) {
+    var textBlob = this.convertStringToBinaryBlob(docSource.text)
+    const fetchOptions = Object.assign({
+      lang: 'lat',
+      textType: 'text',
+      segments: 'singleline'
+    }, tokenizeParams)

```

Comment:
can wait for a later pr but eventually options available should come from the service apidoc

Reply:
Do you mean that we should do everytime an additional request to the remote service for the default settings?
lang and textType are not default
now we support only text,  but `textType` and `lang` are the properties of the source text
But here are the mistake with lang (defined always as lat), I fixed it

segments are not obligatory - I simply leave it here after testing doubline overwrite from the applications parameters. 
Would remove.

Reply:
The user should be able to specify what settings to use for tokenization.  E.g only they know if they are wanting a single segment or multiple, how segments are specified (singe/double line for text, element names for tei, etc)  We may make additional options available as we go. So the client (alignment editor) needs to serve as a proxy to the tokenization service - showing the user the options and passing them on to the service.  I would like not to have the options duplicated in the client, but instead retrieved from the service api doc.

See for example how we presented this sort of thing with the v1 editor in perseids:

![dd75bc68-2552-47f2-8d46-935d9036259e](https://user-images.githubusercontent.com/644870/97711157-18971000-1a93-11eb-9987-412c2e3c3f5f.png)

Reply:
I would it this description to the issue - https://github.com/alpheios-project/alignment-editor-new/issues/32